### ISSUE TITLE
Editor: adjust the intro page in the New Game Wizard, make height higher

### ISSUE THREAD

**Issue by ivan-mogilko** [adventuregamestudio/ags#2622]
After #1903 (after #2574 in 3.6.2) the template selection page has a "Description" field underneath, but since the page height remains the same, the controls are cramped together, which makes it look bad:

<details>
<summary>Click for screenshot</summary>

![image](https://github.com/user-attachments/assets/1785f243-f8fc-4469-8377-99e421587ec5)

</details>

IMHO it needs at least 1 more row for the list of templates, and increase the Description field maybe x1.5 in case the text is longer.

The problem is that the first page of the Wizard contains a bitmap, and if we just increase the dialog height, that will look bad too without image adjustment:
https://raw.githubusercontent.com/adventuregamestudio/ags/refs/heads/master/Editor/AGS.Editor/Resources/wizard.bmp

We need to either adjust this image for another, bigger height, or change the design of this page altogether with some form that will allow us to adjust the window size easier on need.

**Comment by ericoporto** [comment]
The first page of the wizard is in the same control that has everything else. My suggestion would be to move the first page of the wizard to a new control so that the wizard can then load "page controls" inside. Right now it can has both the first page and the "generic page" that supports loading the "page controls" in the same form which makes quite hard to arrange things.

**Comment by ericoporto** [comment]
[ags_bluecup.zip](https://github.com/user-attachments/files/18311612/ags_bluecup.zip)

[Homelightgames from the forums created some awesome resource](https://www.adventuregamestudio.co.uk/forums/editor-development/the-new-look-for-the-new-game-wizard-because-we-need-more-height/msg636668275/#msg636668275), I put it in a zip and attached above.

I think a way to use this would be make the bg stretch as needed (if possible in some way it has some interpolation) and then draw the bluecup in some similar way. I haven't figured how to even make it possible to resize the wizard forms dynamically to test, but this can be done by overriding the paint method of picturebox I think.

**Comment by ivan-mogilko** [comment]
> I think a way to use this would be make the bg stretch as needed (if possible in some way it has some interpolation) and then draw the bluecup in some similar way.

To clarify: do you mean that the bg is stretched, but the cup always drawn in the original size?

**Comment by ericoporto** [comment]
Yeah, so that the cup doesn't stretch, and remains sharp. I don't know if it's possible though.

**Comment by ivan-mogilko** [comment]
> I was trying something, but wanted to test it with that window being resizable, just so I could easily vary the size and see how it would scale, but I couldn't figure how to make that window resizable, afaict there isn't any winforms property in the designer view properties window, but the size of that window is magically fixed.

Is not it FormBorderStyle?

I would like to resolve this issue soon, because 3.6.2 is approaching release candidate imo.
From what I gather, the idea is to move the intro to its own page, in order to make it consistent with other contents. And then figure out how to paint picture background + cup separately so that background scaled but cup remained same size.
